README.md

```diff
@@ -27,21 +41,22 @@ The Extensions API lets you do more without leaving Tableau. Build Tableau exten
 
    **npm start**
 
-5. Launch Tableau and try the sample extensions in a dashboard. The samples are located in the `Samples` folder.
+5. Launch Tableau and try a dashboard sample extension in a dashboard, or a viz extension in a worksheet. The dashboard and viz extension  samples are located in the `Samples` folder.
```

Comment:
Not sure if this matters, but I think there's two spaces after "The dashboard and viz extension (space) (space) samples."

Reply:
Yep. Thanks.

Reply:
done